### DISCUSSION TITLE
How to restrict what members see in Vercel team

### DISCUSSION THREAD

**Discussion by rohanmayya** [vercel/community#3296] [Feature Question] [answered]
### Summary

We have like 4-5 projects on our Vercel team, we want to restrict some members from seeing some projects altogether (just to safeguard things like environment variables on the other projects)

Any suggestions on what can be done here? I went through the "Access Roles" documentation and could not find what I was looking for

### Steps to Reproduce

Add a member to the Vercel team, and try to restrict them from seeing all the projects on the dashboard

**Comment by amyegan** [comment, marked as answer]
Hey @rohanmayya. Access Roles are the right thing to use in general. There is no project-level access control for team members. 

On Pro teams there are [Owner](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#owner-role) and [Member](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#member-role) roles which both allow access to environment variables for all environments. There is also a [Billing](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#billing-role) role which is a free team seat which only has access to manage billing information.

Enterprise teams have additional roles available to them. The [Developer](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#developer-role) role can manage environment variables for development and preview environments only. And [Project Administrators](https://vercel.com/docs/accounts/team-members-and-roles/access-roles#project-administrators) have the same env var restriction as Developers, but with additional access to promote deployments to production and manage other project settings.

**Reply by Filsommer** [reply]
Just wanna say that this is really the thing I dislike most about Vercel. How is it possible that I, on a Pro plan, cannot onboard a new dev to my project, just because you don't allow me to hide certain projects I don't want him to see. REALLY hope you guys fix this urgent issue soon.

**Reply by leerob** [reply]
Hey @Filsommer, thank you for this feedback. While I know this isn't the ideal solution you are looking for, you can also create a separate team for this project.
